backend/routes/contactCreator.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { type User } from '@prisma/client'
+import express from 'express'
+import prisma from '../utils/prismaClient'
+
+const contactCreatorRouter = express.Router()
+const db = prisma
+
+contactCreatorRouter.get('/', async function (req, res) {
+  try {
+    const contacts = await db.contact.findMany({
+      where: { ownerID: (req.user as User).id },
+      include: {
+        relationships: true,
+        importantDateEvent: true
+      }
+    })
```

Comment:
some good reading on some best practices with file structure:

https://blog.devgenius.io/the-three-layer-architecture-for-node-js-applications-ce32a3a30fa6

generally the data access layer should be in its own file and not called directly in the route handlers like we do here

see how hyun did it for Plan https://github.com/wewrap/togetherwewrap/blob/main/backend/routes/planForm.ts

Theres a route file, a controller file, and at least 1 model file that we want to split things up into